opentelemetry-jaeger/src/lib.rs

```diff
@@ -436,12 +436,21 @@ impl Into<jaeger::Log> for api::trace::Event {
             .duration_since(SystemTime::UNIX_EPOCH)
             .unwrap_or_else(|_| Duration::from_secs(0))
             .as_micros() as i64;
-        let mut fields = self
+        let mut event_set_via_attribute = false;
+        let mut fields: std::vec::Vec<jaeger::Tag> = self
```

Comment:
Small nit: this doesn't actually need the explicit type anymore

```suggestion
        let mut fields = self
```